src/Services/Podcasts/Podcast.API/Program.cs

```diff
@@ -37,14 +37,19 @@
 
 app.UseCors();
 
+/*
+    Gets all categories
```

Comment:
* The comment does not really add any value IMHO. 
* IMHO the comment syntax should be consistent with the rest of the code (I can't find any such `/* ... */` comment.) 

In the end I'm not sure what the motivation for this PR is.